frontend/web/styles/3rdParty/_react-diff.scss

```
@@ -1,23 +1,45 @@
-@import "../variables";
+@import '../variables';
+
+// Diff view (react-diff-viewer + Prism for JSON). Colours come from the same
+// tokens as the highlight.js code theme (_hljs.scss), so the diff matches the
+// rest of the app's code snippets and follows light/dark for free.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)